<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Secure</class>
 <widget class="QWidget" name="Secure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>11</number>
   </property>
   <item>
    <widget class="QTabWidget" name="TabWidget3">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>&amp;Security</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>11</number>
       </property>
       <item>
        <widget class="QCheckBox" name="chkAuth">
         <property name="text">
          <string>My &amp;authorization is required before users add me to Contact List</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkHideIP">
         <property name="text">
          <string>Don't show my &amp;IP-address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkWeb">
         <property name="windowTitle">
          <string/>
         </property>
         <property name="text">
          <string>Allow other to view my &amp;status from the Web</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkIgnoreAuth">
         <property name="text">
          <string>I&amp;gnore user authorization</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkUseMD5">
         <property name="text">
          <string>Always use &amp;MD5 authentication</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="grpDirect">
         <property name="title">
          <string>Direct connections</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="btnDirectContact">
            <property name="text">
             <string>Allow direct connection from user listed in &amp;contact list</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="btnDirectAllow">
            <property name="text">
             <string>Allow &amp;direct connection from any user</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="btnDirectAuth">
            <property name="text">
             <string>Allow direct connection from user &amp;upon authorization</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="Spacer6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>&amp;Visible list</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>11</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="lstVisible">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab3">
      <attribute name="title">
       <string>&amp;Invisible list</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>11</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="lstInvisible">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
